src/components/shared/MarkerTooltipContents.js

```diff
@@ -806,10 +806,10 @@ class MarkerTooltipContents extends React.PureComponent<Props> {
         <div className={classNames({ tooltipHeader: details })}>
           <div className="tooltipOneLine">
             <div className="tooltipTiming">
-              {/* tracing markers with no start have a negative start, while the
-                ones with no end have an infinite duration */}
-              {Number.isFinite(marker.dur) && marker.start >= 0
-                ? formatNumber(marker.dur) + 'ms'
+              {/* we don't know the duration if the marker is incomplete (unknown start),
+                  or when the duration is infinite */}
```

Comment:
I liked that the previous comment mentioned that "infinite duration" is when we have no end. Can you add this comment back in some way? Unless we can use "incomplete" for these markers too... That would be nice :)